test/e2e/receive_test.go

```diff
@@ -95,6 +95,152 @@ func TestReceive(t *testing.T) {
 		})
 	})
 
+	t.Run("ha_ingestor_with_ha_prom", func(t *testing.T) {
+		/*
+				The ha_ingestor_with_ha_prom suite represents a configuration of a
+			    naive HA Thanos Receive with HA Prometheus. This is used to exercise
+			    deduplication with external and "internal" TSDB block labels
+
+					 ┌──────┐  ┌──────┬──────┐
```

Comment:
I prefer to try to manually fix this. Exporting things from asciiflow is badly broken since months and makes copying over things super annoying: https://github.com/lewish/asciiflow/issues/253